aea/configurations/base.py

```diff
@@ -1001,6 +1138,10 @@ class AgentConfig(PackageConfiguration):
             "storage_uri",
         ]
     )
+    CHECK_EXCLUDES = [
+        ("private_key_paths",),
+        ("default_routing",),
+    ]
```

Comment:
Why don't we check?

Reply:
cause can contain attributes not mentioned in the current config